src/components/SelectionList/TransactionListItem.tsx

```diff
@@ -0,0 +1,310 @@
+                                    <Icon
+                                        src={Expensicons.Checkmark}
+                                        fill={theme.textLight}
+                                        height={14}
+                                        width={14}
+                                    />
+                                )}
+                            </View>
+                        </PressableWithFeedback>
+                    )}
+                    <View style={[styles.flexRow, styles.flex1, styles.gap3]}>
+                        <View style={[styles.flex1, styles.flexColumn, styles.justifyContentCenter, styles.alignItemsStretch]}>{dateElement}</View>
+                        <View style={[styles.flex1, styles.flexColumn, styles.justifyContentCenter, styles.alignItemsStretch]}>{descriptionElement}</View>
+                        <View style={[styles.flex1, styles.flexColumn, styles.justifyContentCenter, styles.alignItemsStretch]}>
+                            <View style={[styles.flexRow, styles.gap3, styles.flex1, styles.alignItemsCenter]}>
+                                <Avatar
+                                    imageStyles={[styles.alignSelfCenter]}
+                                    size={CONST.AVATAR_SIZE.SMALL}
+                                    source={personalDetails[item.managerID]?.avatar}
+                                    name={personalDetails[item.managerID]?.displayName}
+                                    type={CONST.ICON_TYPE_WORKSPACE}
+                                />
+                                <Text
+                                    numberOfLines={1}
+                                    style={[styles.flex1, styles.flexGrow1, styles.textStrong]}
+                                >
+                                    {personalDetails[item.managerID]?.displayName}
+                                </Text>
+                            </View>
+                        </View>
+                        <View style={[styles.flex1, styles.flexColumn, styles.justifyContentCenter, styles.alignItemsStretch]}>
+                            <View style={[styles.flexRow, styles.gap3, styles.flex1, styles.alignItemsCenter]}>
+                                <Avatar
+                                    imageStyles={[styles.alignSelfCenter]}
+                                    size={CONST.AVATAR_SIZE.SMALL}
+                                    source={personalDetails[item.accountID]?.avatar}
+                                    name={personalDetails[item.accountID]?.displayName}
+                                    type={CONST.ICON_TYPE_WORKSPACE}
+                                />
+                                <Text
+                                    numberOfLines={1}
+                                    style={[styles.flex1, styles.flexGrow1, styles.textStrong]}
+                                >
+                                    {personalDetails[item.accountID]?.displayName}
```

Comment:
`To` is the managerID
```suggestion
                                    {personalDetails[item.managerID]?.displayName}
```